### PR TITLE
Azure pipelines build

### PR DESCRIPTION
**Pull request by danielboth**
#### Pull Request (PR) description
This PR converts xRemoteDesktopSessionHost to use the new release pipeline.

#### This Pull Request (PR) fixes the following issues
None

#### Task list

- [X] Added an entry under the Unreleased section of the change log in the README.md.
      Entry should say what was changed, and how that affects users (if applicable).
- [ ] Resource documentation added/updated in README.md.
- [ ] Resource parameter descriptions added/updated in README.md, schema.mof
      and comment-based help.
- [ ] Comment-based help added/updated.
- [ ] Localization strings added/updated in all localization files as appropriate.
- [ ] Examples appropriately added/updated.
- [X] Unit tests added/updated. See [DSC Resource Testing Guidelines](https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md).
- [ ] Integration tests added/updated (where possible). See [DSC Resource Testing Guidelines](https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md).
- [X] New/changed code adheres to [DSC Resource Style Guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md) and [Best Practices](https://github.com/PowerShell/DscResources/blob/master/BestPractices.md).

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Code of conduct
+
```
Comment (gaelcolas) [inline CODE_OF_CONDUCT.md]: This file as changed actually. Please update to the template ones.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    exclude:
+    - "*-*"
+#variables:
+
+stages:
+  - stage: Build
+    jobs:
+      - job: BuildModuleJob
+        pool:
+          vmImage: 'vs2017-win2016'
```
Comment (gaelcolas) [inline azure-pipelines.yml]: Build stage should be on ubuntu, any reason why not?

File: build.yaml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  Write-Build DarkGray  "$(Get-BuildSynopsis $Task)"
+  "-" * 79
+  Write-Build DarkGray "  $Path"
+  Write-Build DarkGray "  $($Task.InvocationInfo.ScriptName):$($Task.InvocationInfo.ScriptLineNumber)"
+  ""
+
+GitHubConfig:
+  GitHubFilesToAdd:
+    - 'CHANGELOG.md'
+  GitHubConfigUserName: gaelcolas
```
Comment (gaelcolas) [inline build.yaml]: This has changed in the template, please update
dsccommunity
dsccommunity@outlook.com

File: source/xRemoteDesktopSessionHost.psd1 [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            # A URL to the license for this module.
+            LicenseUri = 'https://github.com/dsccommunity/xRemoteDesktopSessionHost/blob/master/LICENSE'
+
+            # Set to a prerelease string value if the release should be a prerelease.
+            Prerelease = ''
+
+            # A URL to the main website for this project.
+            ProjectUri = 'https://github.com/dsccommunity/xRemoteDesktopSessionHost'
+
+            # A URL to an icon representing this module.
```
Comment (gaelcolas) [inline source/xRemoteDesktopSessionHost.psd1]: You might want to remove this as you added it above.

Comment (danielboth) [reply source/xRemoteDesktopSessionHost.psd1]: Not sure what you meant by this? The only thing above is the LicenseUri in this file, having ProjectUri in the metadata seems valid to me?

Comment (gaelcolas) [reply source/xRemoteDesktopSessionHost.psd1]: I meant the comment line 62/63. No biggie though.

**Pull request merged**